components/esm/loadcell.hpp

```diff
@@ -90,6 +90,8 @@ struct Cell
 
   Cell() : mName(""),
            mRegion(""),
+           mData(DATAstruct()),
```

Comment:
Again, the DATAstruct() part looks redundant.

Reply:
Again, it is not redundant. It allows to make sure that all fiends of mData struct are initialided by zeroes. With our current behaviour mData contains old values since C++ does not fill memory by zeroes automatically. At least, GCC7 behaves in this way.

Reply:
What's wrong with DATAstruct? It is just a simple structure with three fields.